backend/executor.go

```diff
@@ -28,17 +29,20 @@ var errShuttingDown error = status.Error(codes.Canceled, "shutting down")
 
 type ExecutionResults struct {
 	Response *protos.OrchestratorResponse
-	complete chan interface{}
+	complete chan struct{}
```

Comment:
@ItalyPaleAle What's the difference between `interface{}` and `struct{}` for channels? I was using the former since that's what the books on Go were telling me to use, but not sure if that guidance is outdated.

Reply:
It's admittedly a micro-optimization, but `struct{}` requires no memory. https://stackoverflow.com/questions/52035390/why-using-chan-struct-when-wait-something-done-not-chan-interface